posts/2022-05-05-real-time-vs-analytical-data.md

```diff
@@ -0,0 +1,100 @@
+
+Let's look at two ways it's being used in a little more detail:
+
+#### E-commerce and personalization
+
+E-commerce companies have learned that real-time personalized offers and recommendations lead to improved digital experiences for their customers and boosted sales for themselves. At any moment, a company can analyze its customer's data, previous interactions, current behavior, and more to modify the copy, images, promotions, and prices the customer sees on screen. Then, based on a combination of historical and current data, at checkout, customers can be presented with additional items that might make sense to add to their cart. Real-time analytics drive this ability to provide customers with the right information at just the right time.
+
+#### Security risk monitoring and fraud detection
+
+Real-time analytics are a crucial weapon in the ever-escalating war against hackers and fraudsters. Security breaches can be crippling for a company's bottom line and reputation. Customers can quickly lose trust and flee to a competitor. Luckily, real-time analysis of a large volume of historic and live data can be used to identify patterns and anomalies, so companies can be proactively alerted to potential risks and breaches. The monitoring is continuous and can include user profile, environment, geographic, and biometric data, as well as other data sources.
+
+## Real-time data and streaming data
+
+Streaming data is another aptly named technology. In this case "streaming" refers to a flow of data, from a variety of sources, that is continuously generated and delivered. Each data point represents an event that happened at a particular time. That's why streaming data is often called event data. Streaming data is typically used in OLTP systems. In these environments, real-time stream processing technology is used to store, process, analyze, and act upon incoming data streams, at the very moment they're created. This is in stark contrast to batch processing where these functions have to be completed separately before data can be used.
+
+## Next steps: Learn how to extend the value of real-time data
+
+We hope this helped unravel the subtle differences between real-time data, operational data, analytical data, and streaming data, along with illustrating how important they all are to the success of modern organizations. While it comes in a few flavors, a couple things are clear—real-time data is a key driver of success for modern organizations and it's responsible for many of the conveniences we enjoy every day. Whether it's real-time operational data driving e-commerce transactions or real-time analytical data that companies use to make their most important decisions, real-time data is the critical, irreplaceable element. 
+
+Real-time data can become even more valuable when it's shared and integrated across companies, clouds, and SaaS applications. Learn how that's now possible with a real-time data lake.
```

Comment:
Learn more about real-time data sharing in [What is Real-Time Data Sharing](https://www.vendia.net/blog/what-is-real-time-data-sharing)?